projects/step-core/src/lib/modules/table/components/table/table.component.scss

```diff
@@ -1,46 +1,55 @@
+@use '../../../../../../styles/core-mixins' as core;
+
+@include core.table-space-between-columns;
+
 :host {
   position: relative;
   display: block;
-}
 
-table {
-  width: 100%;
-}
+  table {
+    width: 100%;
+    border-spacing: 10px 0 !important;
+  }
 
-tr.mat-row:hover {
-  background: var(--darken-background-10);
-}
+  ::ng-deep .mat-form-field-infix {
+    width: auto !important;
+  }
 
-th > mat-form-field {
-  width: 90%;
-}
+  ::ng-deep tr > td.mat-cell:not(.mat-column-bulkSelection):not(.mat-column-actions) {
+    padding: 0 8px;
+  }
 
-.spinner-container {
-  position: absolute;
-  z-index: 1;
-  width: 100px;
-  height: 100px;
-  left: calc(50% - 50px);
-  top: calc(50% - 50px);
-}
+  tr.mat-row:hover {
+    background: var(--darken-background-10);
+  }
 
-.additional-header {
-  padding: 0 24px;
-}
+  .spinner-container {
+    position: absolute;
+    z-index: 1;
+    width: 100px;
+    height: 100px;
+    left: calc(50% - 50px);
+    top: calc(50% - 50px);
+  }
 
-.no-data {
-  text-align: center;
-}
+  .additional-header {
+    padding: 0 24px;
+  }
 
-mat-paginator {
-  border-radius: 0 0 16px 16px;
-}
+  .no-data {
+    text-align: center;
+  }
 
-:host tr {
-  ::ng-deep step-btn-group {
-    opacity: 0.5;
+  mat-paginator {
+    border-radius: 0 0 16px 16px;
   }
-  &:hover ::ng-deep step-btn-group {
-    opacity: 1;
+
+  tr {
+    ::ng-deep step-btn-group {
```

Comment:
ng-deep usage

Reply:
This is encapsuled in :host so it is fine for now. When we have more time we should take a better look what to do of ng-deeps in the project.